### DISCUSSION TITLE
TOTP App which one? Advice?

### DISCUSSION THREAD

**Discussion by Alex-e107nl** [e107inc/e107#5132] [General]
Well just start using Github and already comes question:
_Your account meets this criteria, and you will need to enroll in 2FA within 45 days, by January 17th, 2024 at 00:00 (UTC). After this date, your access to GitHub.com will be limited until you enroll in 2FA. Enrolling is easy, and we support several options, starting with TOTP apps and text messages (SMS) and then adding on passkeys and the GitHub Mobile app._

Who has good advice on which app i should use, i work with win11 Android and Apple ..

I think there are several options or should i just use one on this page [https://github.com/e107inc/twofactorauth/#recommended-authenticator-applications](https://github.com/e107inc/twofactorauth/#recommended-authenticator-applications) Thx @Moc  :-)

It has to work with this plugin of course...

**Comment by Moc** [comment]
You can use any you like :) I use Authy myself. 

And, you're welcome. Let me know if you have any feedback by creating an issue in the repository.

**Reply by Jimmi08** [reply]
Good to know. I have there SMS/Text settings.  In fact I had no idea what I was doing and this option looked "so normal".

**Reply by Moc** [reply]
For Github you can also use the Github mobile app. I use Authy for other platforms too.

**Comment by Alex-e107nl** [comment]
Oke, had some problems with authy but they are solved, i have installed 2factauth on e107.nl, translated it and tested it. Works fantastic Moc, THX! ( [https://github.com/Alex-e107nl/twofactorauth](https://github.com/Alex-e107nl/twofactorauth) )

**Reply by Moc** [reply]
@Alex-e107nl Great! Feel free to submit your translation as a pull-request, then I'll merge it in.

**Reply by Moc** [reply]
And feel free to submit any new feature requests (in the issue tracker in the repository).

**Reply by Alex-e107nl** [reply]
hmmm I thought I already did that :-) it's all still a bit new to me....

**Comment by nmontalva** [comment]
What are good *desktop* options of TOTP? I been part of the resistance for so long, I am not going to use a smartphone now.

**Reply by Alex-e107nl** [reply]
Maybe also [https://authy.com/download/](https://authy.com/download/)? The app on android works fine, have to test the desktop version also.
